scripts/copilot/instructions.md

```
@@ -0,0 +1,1456 @@
+  - `--nutui-font-weight-medium` (用途: $font-weight-medium)
+  - `--nutui-font-weight-bold` (用途: $font-weight-bold)
+
+## 2. 组件 API 智能参考 (Component Dictionary)
+
+以下是 NutUI-React 核心组件的参数规范，补全时请参考中文描述：
+
+### <Button />
+
+| 属性 (Prop) | 说明 (Description) | 类型 (Type) | 默认值 (Default) |
+| --- | --- | --- | --- |
+| type | 按钮的样式 | ``default` | `primary` | `info` | `warning` | `danger` | `success`` | default |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 9; Too many cells, extra data will be missing

(MD056, table-column-count)